pages/api/db/prisma.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from '@prisma/client';
```

Comment:
@leewan-09 i was curious if we could move this file to /lib
any special reason for keeping it inside api/ folder?

Reply:
we can sure put it outside API folder I think there would not be any problem with that.

Reply:
i think /lib folder would be best for it

Reply:
We could move but it is our DB connection file I would like to keep it like that.

Reply:
yup, we can go on with it @leewan-09 try to put it on /lib and insure everything is working fine as expected .

Reply:
> We could move but it is our DB connection file I would like to keep it like that.

yes I don't think it's a good idea to put it inside API folder

Reply:
From [here](https://www.youtube.com/c/TheoBrowne1017) i learnt that...

Reply:
yup, it's just a naming convention everyone can have different opinions on it. My main reason to say this is that the API folder is going to have too many routes and we already have an /lib folder I guess so it's no harm putting it that way.